collectors/apps.plugin/apps_plugin.c

```diff
@@ -107,6 +107,26 @@ static int
 static char *user_config_dir = CONFIG_DIR;
 static char *stock_config_dir = LIBCONFIG_DIR;
 
+// some variables for keeping track of processes count by states
+#define NETDATA_CHART_PRIO_SYSTEM_PROCESS_STATES 601
+typedef enum {
+    RUNNING = 0,
+    SLEEPING, // interruptible sleep
+    SLEEPING_D, // uninterruptible sleep
+    ZOMBIE,
+    STOPPED,
+    END,
```

Comment:
A suggestion: Rename these definitions, for something more specific like `APPS_STATUS_RUNNIG`, `APPS_STATUS_SLEEPING`, and so on.

Reply:
good suggestion